libs/sdk-bindings/src/breez_sdk.udl

```diff
@@ -733,10 +764,21 @@ dictionary SendOnchainRequest {
     u32 sat_per_vbyte;
 };
 
+dictionary PayOnchainRequest {
+    u64 send_amount_sat;
+    u64 receive_amount_sat;
```

Comment:
How do we validate that the receive_amount matches the send_amount in a way that they were generated in a valid prepare request? Do we leave that to the user responsibility? Even if this is the case we still need to ensure we won't create eventually illegal claim transaction.

Reply:
How about this: we could check if `receive_amount_sat > onchain_amount_sat`. Even if it's a 1 sat difference, it's technically a valid fee. Or we could enforce a minmum claim feerate of 1 sat/vB.

We can validate this in `create_reverse_swap`.

What do you think?

Reply:
I think going with at least 1 sat/vB for the claim is good.

Reply:
Done: c5c9e9208492181760ba83a16cccd757bffdd222